### PR TITLE
IDE-3113 move addmarker to MigrateProjectHandler and remove Workspace…

### PR DESCRIPTION
**Pull request by AndyWu2015**
…MigrationImpl

### CODE REVIEW

File: tools/plugins/com.liferay.ide.project.ui/src/com/liferay/ide/project/ui/migration/MigrateProjectHandler.java [hunk 1]
```diff
@@ -456,11 +460,35 @@ private boolean shouldShowMessageDialog( IProject project )
         return markers != null && markers.length > 0;
     }
 
+    @SuppressWarnings( "deprecation" )
     private boolean shouldAdd( Problem problem )
     {
-        String path = problem.getFile().getAbsolutePath().replaceAll( "\\\\", "/" );
+        File file = problem.getFile();
```
Comment (gamerson) [inline tools/plugins/com.liferay.ide.project.ui/src/com/liferay/ide/project/ui/migration/MigrateProjectHandler.java]: Lets change this code to use the new 'find a target' code that you changed in another pull request.

Comment (gamerson) [inline tools/plugins/com.liferay.ide.project.ui/src/com/liferay/ide/project/ui/migration/MigrateProjectHandler.java]: The one from here: https://github.com/gamerson/liferay-ide/pull/2035

Comment (AndyWu2015) [inline tools/plugins/com.liferay.ide.project.ui/src/com/liferay/ide/project/ui/migration/MigrateProjectHandler.java]: you mean reuse that code ?

Comment (AndyWu2015) [inline tools/plugins/com.liferay.ide.project.ui/src/com/liferay/ide/project/ui/migration/MigrateProjectHandler.java]: done

**Comment by gamerson** [comment]
Merged, thanks! See 320c387...1a09b82 <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

**Pull request merged**